examples/storybook/src/stories/RefinementList.stories.ts

```diff
@@ -64,7 +65,9 @@ storiesOf('RefinementList', module)
         </ais-panel>
       `,
       methods: {
-        transformItems: items =>
+        transformItems: (
+          items: (RefinementListItem & { highlighted: string })[]
```

Comment:
refinement list item doesn't already have highlighted?

Reply:
It was not there yet in 4.7, I'll attempt an update

Reply:
Actually there are too many breaking changes in types, can't make the update smooth. If you don't mind I'll keep this PR about updating to Angular 12 than make another one where i'll update everything (SearchClient, IS and types simultaneously in Angular IS source and examples)

Reply:
makes sense :)